File: aliyun-java-best-practise/src/main/java/com/aliyun/bestpractise/RunInstancesDemo.java
package com.aliyun.bestpractise;

import com.aliyuncs.DefaultAcsClient;
import com.aliyuncs.IAcsClient;
import com.aliyuncs.ecs.model.v20140526.RunInstancesRequest;
import com.aliyuncs.ecs.model.v20140526.RunInstancesResponse;
import com.aliyuncs.exceptions.ClientException;
import com.aliyuncs.profile.DefaultProfile;

public class RunInstancesDemo {
    public static void main(String[] args) {
        DefaultProfile profile = DefaultProfile.getProfile(
                "RegionId",
                "accessKeyId",
                "accessKeySecret");
        IAcsClient client = new DefaultAcsClient(profile);
        RunInstancesRequest request = new RunInstancesRequest();
        request.setLaunchTemplateId("创建实例的模板ID");
        RunInstancesResponse response;
        try {
            response = client.getAcsResponse(request);
        } catch (ClientException e) {
            e.printStackTrace();
        }
    }
}

File: aliyun-python-best-practise/aliyunbestpractise/runInstances_demo.py
from aliyunsdkcore.client import AcsClient
from aliyunsdkecs.request.v20140526.RunInstancesRequest import RunInstancesRequest
client = AcsClient('access_key_id', 'access_key_secret', 'region_id')
request = RunInstancesRequest()
request.set_LaunchTemplateId('创建实例的模板ID')
response = client.do_action_with_exception(request)
print(response)
